File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/integrasjoner/kafka/KafkaTopicKey.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.integrasjoner.kafka;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class KafkaTopicKey {
    String orgnr;
    int kvartal;
    int årstall;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/integrasjoner/kafka/dto/KafkaTopicKey.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.integrasjoner.kafka.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class KafkaTopicKey {
  String orgnr;
  int kvartal;
  int årstall;
}
